content/artisan-static.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Artisan Static"
+github: https://github.com/raniesantos/artisan-static
+author: raniesantos
+demo: https://artisan-static-demo.netlify.com/
+date: 2020-06-03
+ssg:
+    - Jigsaw
```

Comment:
SSG jigsaw will need to be added to the ssg taxonomy

Reply:
Fixed